suitcase/tests/conftest.py

```diff
@@ -1,30 +1,43 @@
 import uuid
 import pytest
-from metadatastore.mds import MDS
-from databroker.broker import Broker
+from databroker import Broker
 import os
 
 AUTH = os.environ.get('MDSTESTWITHAUTH', False)
 
 
+test_config = {
+    'metadatastore': {
+        'module': 'databroker.headersource.mongo',
+        'class': 'MDS',
+        'config': {
+            'host': 'localhost',
+            'port': 27017,
+            'database': 'test1',
+            'timezone': 'US/Eastern'}
+    },
+    'assets': {
+        'module': 'databroker.assets.mongo',
+        'class': 'Registry',
+        'config': {
+            'host': 'localhost',
+            'port': 27017,
+            'database': 'test2'}
+    }
+}
+
+
 @pytest.fixture(params=[1], scope='function')
 def db_all(request):
```

Comment:
It is probably better to import the fixtures from databroker here rather than re-writing them.  See what we do in bluesky.